plugins/snmp_receiver_plugin/build/config/snmp.conf

```diff
@@ -1,17 +1,16 @@
 [SNMP]
 # SNMP configuration options
-snmp_ip=0.0.0.0
-snmp_port=162
-community=public
-multiple_events=False
+snmp_port=162 # port to listen for traps
```

Comment:
please move all the comments / explanations to be on the line before the field...
it will be more clear and less crowded...